firefox_puppeteer/tests/test_windows.py

```diff
@@ -77,11 +77,14 @@ def test_base_window_basics(self):
                           win1.send_shortcut, 'l', acel=True)
 
     def test_base_window_open_close(self):
+        def setUp(self):
+            FirefoxTestCase.setUp(self)
+
         # force BaseWindow instance
         win1 = BaseWindow(lambda: self.marionette, self.browser.handle)
 
         # Open a new window (will be focused), and check states
-        win2 = win1.open_window()
+        win2 = self.browser.open_browser(trigger='menu')
```

Comment:
Please don't change this line but only the test for the browser window. BaseWindow does not support any kind of trigger beside a callback.